### PR TITLE
MINOR Move some things into committer-tools

### PR DESCRIPTION
**Pull request by mumrah**
Move reviewers.py and approve-workflows.py into committer-tools. Also update the README

### CODE REVIEW

**Comment by chia7712** [comment]
Should `kafka-merge-pr.py` be included?

https://github.com/apache/kafka/blob/trunk/kafka-merge-pr.py

**Comment by mumrah** [comment]
I've also included a new find-unfinished-test.py which I've used in the past to find hanging tests

File: committer-tools/README.md [hunk 1]
```diff
@@ -63,8 +81,37 @@ export GITHUB_TOKEN="$(gh auth token)"
...
+
+## Find Hanging Tests
+
+This script is used to infer hanging tests from the Gradle output. It looks for
+tests that were STARTED but do not have a corresponding FINISHED or FAILED.
+
+Usage:
+
+```bash
+python2 find-unfinished-test.py ~/Downloads/logs_28218821016/5_build\ _\ JUnit\ tests\ Java\ 11.txt
```
Comment (chia7712) [inline committer-tools/README.md]: out of curiosity, why using `python2` rather than `python3`?

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**